src/components/funnels/funnels/register/steps/ticket/level.tsx

```diff
@@ -51,9 +49,10 @@ const TicketLevel = (_: ReadonlyRouteComponentProps) => {
 								<TicketLevelCard
 									id={id}
 									price={prices[ticketType.type]}
-									expirationDate={expirationDate}
 									label="Ticket level"
 									priceLabel="A ticket"
+									priceLabelDay="A day ticket"
```

Comment:
Could you please undo the priceLabelDay and isDayTicket properties here, it's better to do what I mention below in en.ftl